src/App/App.js

```diff
@@ -20,6 +20,12 @@ const App = () => {
     "100000020",
     "100000021",
   ];
+  return cardlist.sort(() => 0.5 - Math.random());
```

Comment:
Só adicionando uma linha aqui antes do retorno

```suggestion

  return cardlist.sort(() => 0.5 - Math.random());
```